### PR TITLE
Fix onError type cast error.

### PR DESCRIPTION
**Pull request by isoos**
Found in https://github.com/dart-lang/pub-dartlang-dart/issues/1547

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: memcache
-version: 0.3.0+1
+version: 0.3.1
```
Comment (kevmoo) [inline pubspec.yaml]: Couldn't this be 0.3.0+2?

**Review by kevmoo — APPROVED** [review]

**Comment by isoos** [comment]
The commit got picked up, but I'm still not sure why CI fails.

**Comment by kevmoo** [comment]
I hit restart on the job – this one is often flaky

**Comment by kevmoo** [comment]
I'll land this anding a retry on these...

**Comment by kevmoo** [comment]
published and tagged - 0.3.0+2

**Pull request merged**